web/modules/custom/dpl_library_agency/src/Form/GeneralSettingsForm.php

```diff
@@ -156,7 +157,14 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#type' => 'url',
       '#title' => $this->t('Ereolen link', [], ['context' => 'Library Agency Configuration']),
       '#description' => $this->t('My page in ereolen', [], ['context' => 'Library Agency Configuration']),
-      '#default_value' => $config->get('ereolen_my_page_url'),
+      '#default_value' => $config->get('ereolen_my_page_url') ?? self::EREOLEN_MY_PAGE_URL,
+    ];
+
+    $form['reservations']['ereolen_homepage_url'] = [
+      '#type' => 'url',
+      '#title' => $this->t('Ereolen home link', [], ['context' => 'Library Agency Configuration']),
+      '#description' => $this->t('Home page in ereolen', [], ['context' => 'Library Agency Configuration']),
+      '#default_value' => $config->get('ereolen_homepage_url') ?? self::EREOLEN_HOMEPAGE_URL,
```

Comment:
Please consider the configuration here.

I see that it is the responsibility of the CMS to inject these urls but:

1. Is it necessary that they are configurable? I am not sure that Ereolen changes url structures very often.
2. If they are to be configurable I suggest that we provide a sensible default for these values based on the current version of the site.

Reply:
I also have a small comment, but this is regarding the translations in:
web/profiles/dpl_cms/translations/da.po

Should the new strings not be added and translated here?